### ISSUE TITLE
Release v0.9 based on libzim 9.3.0

### ISSUE THREAD

**Issue by Jaifroid** [openzim/javascript-libzim#72]
As there have been a few important changes in libzim 9.1.0, especially dealing with Zimit 2 archives, we should release a new version based on 9.1.0.

**Comment by Jaifroid** [comment]
We need to update to the latest, which is now libzim 9.3.0. This should ensure we don't experience errors with the latest ZIM types that have no v0 title index.

**Comment by Jaifroid** [comment]
Blocked by https://github.com/openzim/libzim/issues/973

**Comment by Jaifroid** [comment]
In the meantime, we could release v9.2.3...

**Comment by Jaifroid** [comment]
v9.2.3 was released as https://github.com/openzim/javascript-libzim/releases/tag/v0.7

https://github.com/kiwix/kiwix-build/issues/820 is now unblocked. Waiting for nightly cron to test.

**Comment by kelson42** [comment]
The resolution of this issue should be unblocked now. See https://download.openzim.org/release/libzim/libzim_wasm-emscripten-9.3.0-2.tar.gz

**Comment by Jaifroid** [comment]
@kelson42 Indeed, compiling and testing the release package to WASM/ASM [are now passing](https://github.com/openzim/javascript-libzim/actions/runs/15331731777/job/43235117209). Thank you very much!

This issue will remain open until I've released v0.8 based on the new build, which I aim to do soon.
